vtex-apps/bazaarvoice
# Feature/add comments

#### What is the purpose of this pull request?
Responses to Reviews were not being displayed.  Added the ability to display ClientResponses

#### What problem is this solving?
Merchant requested ability to display ClientResponses

#### How should this be manually tested?
Enable Show Client Responses in app settings
Verify that Client Responses are shown on Reviews

#### Screenshots or example usage
![image](https://user-images.githubusercontent.com/47258865/92729288-1c829d80-f340-11ea-803e-01cebcd06f47.png)


#### Types of changes
- [X ] New feature (a non-breaking change which adds functionality)
